eth2/beacon/helpers.py

```diff
@@ -165,6 +165,13 @@ def get_active_validator_indices(validators: Sequence['ValidatorRecord'],
     )
 
 
+#
+# Get the epoch that a slot is in
+#
+def epoch_from_slot(slot: int, epoch_length: int) -> int:
```

Comment:
We use `SlotNumber` for slot number type.
```suggestion
def epoch_from_slot(slot: SlotNumber, epoch_length: int) -> int:
```